packages/http-client/src/__tests__/ceramic-http-client.test.ts

```diff
@@ -15,11 +30,81 @@ const initial = {
   ],
 } as unknown as StreamState
 
+const GET_RESPONSE = {
+  ...initial,
+  state: initial,
+}
+
+let did: DID
+let getDidFn
+let client: CeramicClient
+
+beforeEach(() => {
+  jest.resetAllMocks()
+})
+
 describe('buildStreamFromState', () => {
   test('build instance of Streamtype', async () => {
```

Comment:
this test feels redundant now of the 	`build instances of Streamtype from URLs with extensions` test you added below.  You can probably just delete this test (and the API_URL constant which then won't be used anymore)